docs/pagination.md

```diff
@@ -0,0 +1,71 @@
+# Pagination
+
+Pagination is supported using the metadata support that is built into Ember Data. The
+metadata from Django REST Framework paginated list views is updated on every request
+to sever.
+
+
+## Retrieving the Metadata
+
+To get a page of records simply run a find request with the `page` query param:
+
+```js
+var result = this.store.find("post", {
+  page: 1
+});
+```
+
+It's possible to override the server-side page size by also including the `page_size`
```

Comment:
Given that page_size is off by default in DRF, I think we should add a pageSize property to the metadata. It's difficult for applications to get the page size but it's easy to do in the serializer. The page size is needed to calculate the number of pages. What do you think?